packages/artifacts/package.json

```diff
@@ -22,7 +22,6 @@
   },
   "dependencies": {
     "@aws-sdk/client-s3": "^3.414.0",
-    "circomlib": "^2.0.5",
```

Comment:
I think this should stay there

Reply:
I've approved but I do agree with this comment!